src/content/docs/logs/ui-data/obfuscation-ui.mdx

```diff
@@ -3,25 +3,29 @@ title: Hash or mask sensitive data in your logs (Obfuscation)
 metaDescription: 'How to hash or mask sensitive data in logs you send to New Relic with our obfuscation UI or API.'
 ---
 
-After logs have been shipped to New Relic, any sensitive information in the logs can be **obfuscated** before being stored in NRDB by using obfuscation rules.
+After logs have been shipped to New Relic, any sensitive information in the logs can be obfuscated before being stored in [NRDB](/docs/data-apis/get-started/nrdb-horsepower-under-hood/), our New Relic database, by using obfuscation rules.
 
-Sensitive information might include personally-identifiable information such as credit card numbers or other data that you may be required by regulation to protect when stored.
+Sensitive information might include personally-identifiable information such as credit card numbers, Social Security numbers, national IDs, or other data that you may be required by regulation to protect when stored.
 
 You can define regular expressions matching your sensitive information, and then create rules to obfuscate that data. You can choose either to have sensitive information masked or hashed.
 
+<Callout variant="tip">
+  Obfuscation has per-minute CPU limits for obfuscation. If your account hits your resource limits, logs won't be obfuscated as expected. To easily check your CPU limits, go to your [system **Limits** page](/docs/data-apis/manage-data/view-system-limits/#limits-ui) in the New Relic Data Management UI.
```

Comment:
```suggestion
  Obfuscation has per-minute CPU limits. If your account hits your resource limits, logs won't be obfuscated as expected. To easily check your CPU limits, go to your [system **Limits** page](/docs/data-apis/manage-data/view-system-limits/#limits-ui) in the New Relic Data Management UI.
```

"Obfuscation has per-minute CPU limits for obfuscation" sounds redundant. I deleted 'for obfuscation,' but if you think there needs to be a clarifying clause, something like "Obfuscation has a per-minute CPU limit for number of items that need to be masked/hashed/whatever"

Reply:
oops! thanks for catching that!